spaceship/lib/spaceship/tunes/build.rb

```diff
@@ -173,29 +174,11 @@ def submit_for_beta_review!(metadata)
           build_number: self.build_version,
           platform: self.platform,
 
-          # Required Metadata:
-          changelog: "No changelog provided",
-          description: "No app description provided",
-          feedback_email: "contact@company.com",
-          marketing_url: "http://marketing.com",
-          first_name: "Felix",
-          last_name: "Krause",
-          review_email: "contact@company.com",
-          phone_number: "0123456789",
-          significant_change: false,
-
-          # Optional Metadata:
-          privacy_policy_url: nil,
-          review_user_name: nil,
-          review_password: nil,
-          encryption: false
+          review_notes: "Not provided" # this seems to be required, even though it is specified as optional
         }.merge(metadata)
 
         client.submit_testflight_build_for_review!(parameters)
 
-        # Last, enable beta testing for this train (per iTC requirement). This will fail until the app has been approved for beta testing
-        self.build_train.update_testing_status!(true, 'external', self)
-
```

Comment:
I wanted to check that this was ok? The comment suggests this is required, but for me it just meant a failure since it had just been submitted for review, so obviously couldn't then be distributed.